dashboard/config/levels/custom/dance/New Dance Lab Project.level

```diff
@@ -48,23 +48,23 @@
     "mini_rubric": "false",
     "validation_enabled": "false",
     "hide_pause_button": "false",
+    "blockly_variables": "dancer1",
```

Comment:
is there any risk with adding these new fields? I'm guessing they are new defaults, but we want to be careful editing the new project level.

Reply:
This field, and some others that look new are actually just in new positions. See line 55 (old version) below.

Reply:
oh thanks! reviewing too fast 😅